railties/lib/rails/generators/rails/app/templates/config/initializers/content_security_policy.rb.tt

```diff
@@ -11,6 +11,10 @@
 #   policy.object_src  :none
 #   policy.script_src  :self, :https
 #   policy.style_src   :self, :https
+<%- unless options[:skip_javascript] -%>
```

Comment:
I think `<%- if webpack_install? -%>` may be a better option to check for this.

Reply:
:+1:

Reply:
`webpack_install?` returns true when specified `skip-webpack-install`.

When the `skip-webpacker-install` option is specified, it means that `webpacker` is used, but `webpacker:install` is not executed during `rails new`.
In this case `webpacker` will be used, so this comment should be. Therefore, `webpack_install?` can not be used for checking here.